aspnetcore/fundamentals/middleware/index/snapshot/Chain/StartupUseWhen.cs

```diff
@@ -0,0 +1,31 @@
+﻿public class Startup
+{
+    private readonly ILogger<Startup> _logger;
+
+    public Startup(ILogger<Startup> logger)
```

Comment:
I'm taking advantage of the fact that this is 2.2 and injecting a logger into the constructor.